pkg/chartverifier/profiles/profile.go

```diff
@@ -0,0 +1,100 @@
+package profiles
+
+import (
+	"errors"
+	"github.com/redhat-certification/chart-verifier/pkg/chartverifier/checks"
+	"gopkg.in/yaml.v3"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"runtime"
+)
+
+type Annotation string
+
+const (
+	DigestAnnotation                 Annotation = "Digest"
+	OCPVersionAnnotation             Annotation = "OCPVersion"
+	LastCertifiedTimestampAnnotation Annotation = "LastCertifiedTimestamp"
+)
+
+type Profile struct {
+	Apiversion  string          `json:"apiversion" yaml:"apiversion"`
+	Kind        string          `json:"kind" yaml:"kind"`
+	Name        string          `json:"name" yaml:"name"`
+	Annotations []Annotation    `json:"annotations" yaml:"annotations"`
+	Checks      []*ProfileCheck `json:"checks" yaml:"checks"`
+}
+
+type ProfileCheck struct {
```

Comment:
The `ProfileCheck` type already lives in the `profiles` package, so the `Profile` prefix can be dropped as well.

Reply:
Done. Managed to squash the previous commit with this oneso if you look at the commit you will see other changes - oops.